dht_test.go

```diff
@@ -854,83 +854,6 @@ func TestRefreshBelowMinRTThreshold(t *testing.T) {
 	assert.Equal(t, dhtE.self, dhtA.routingTable.Find(dhtE.self), "A's routing table should have peer E!")
 }
 
-// Check to make sure we re-fill the routing table from connected peers when it
-// completely empties.
-func TestEmptyTable(t *testing.T) {
```

Comment:
Do we have a replacement test? It doesn't have to be "on empty", but we should have some form of "my peers are dropping like flies, I need to look for more peers" test.

Reply:
@Stebalien Take a look at `TestRefreshBelowMinRTThreshold`. Let me know if you  think we still need another test.